lib/jekyll-admin/apiable.rb

```diff
@@ -38,20 +37,26 @@ def to_api(include_content: false, include_documents: false)
       output.delete("output")
 
       # By default, calling to_liquid on a collection will return a docs
-      # array with each rendered document, which we don't want. Instead
-      # include our own documents array, without the content
+      # array with each rendered document, which we don't want.
       if is_a?(Jekyll::Collection)
         output.delete("docs")
-        if include_documents
-          output["documents"] = docs.sort_by(&:date).reverse.map(&:to_api)
-        end
+      end
+
+      if is_a?(Jekyll::Document)
+        output["dir"] = File.join("/", relative_path_from_source.dirname.to_s)
```

Comment:
Yeap, but `relative_path` includes `filename` while `dir` does not. However, I thought I'd need this in the front end but I didn't. I'm removing this field.